p4proto/p4src/linux_networking/lnw_ct.p4

```diff
@@ -1,3 +1,19 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ // SPDX-License-Identifier: Apache-2.0
```

Comment:
will be good to have Identifier right below Copyright line, something like this: 
* Copyright (c) 2021-2022 Intel Corporation.
* SPDX-License-Identifier: Apache-2.0

Reply:
I agree. The SPDX ID should be easy to find.

Reply:
Fixed. Addressed all the issues.